crates/ruff/src/rules/flake8_simplify/rules/ast_if.rs

```diff
@@ -92,6 +95,33 @@ impl Violation for UseTernaryOperator {
     }
 }
 
+define_violation!(
+    /// ### What it does
+    /// Checks if consecutive `if` branches have the same body.
+    ///
+    /// ### Why is this bad?
+    /// These branches can be combine using the python `or` statement
+    ///
+    /// ### Example
+    /// ```if x = 1:
+    ///     print("Hello")
+    /// elif x = 2:
+    ///     print("Hello")
+    /// ```
+    ///
+    /// Use instead:
+    /// ```if x = 1 or x = 2
```

Comment:
```suggestion
    /// ```python
    /// if x = 1:
    ///     print("Hello")
    /// elif x = 2:
    ///     print("Hello")
    /// ```
    ///
    /// Use instead:
    /// ```python
    /// if x = 1 or x = 2
```